src/dashboard/Push/PushNew.react.js

```diff
@@ -203,6 +191,17 @@ export default class PushNew extends DashboardView {
     }
 
     const push_time = extractPushTime(changes);
+
+    // Gather the translations, and inject into the payload
+    Object.keys(changes).forEach((key) => {
+      // translations are stored as `tranlation[lang]` strings as keys,
+      // this is why we slice it this way
```

Comment:
Clear.  thanks...just thinking out loud, not asking for change....

```
const needle = 'translation[';
if (key.indexOf(needle) === 0) {
  const locale = key.slice(needle.length, key.length - 1);
```

Reply:
Eheh, if you need a comment, many others may need a comment. it's better with it than without.